homeassistant/components/yale/__init__.py

```diff
@@ -35,7 +35,7 @@ async def async_setup_entry(hass: HomeAssistant, entry: YaleConfigEntry) -> bool
                 hass, entry
             )
         )
-    except ValueError as err:
+    except config_entry_oauth2_flow.ImplementationUnavailableError as err:
```

Comment:
```suggestion
    except ImplementationUnavailableError as err:
```
let's import it, so it's a bit more readable